docs/destinations/forwarding-events/elasticsearch/index.md

```diff
@@ -217,7 +217,7 @@ The sink is configured using a HOCON file, for which you can find examples [her
 | output.good.cluster.documentType | Optional. The Elasticsearch index type. Index types are deprecated in ES >=7.x Therefore, it shouldn't be set with ES >=7.x |
 | output.good.chunk.byteLimit | Optional. Bulk request to Elasticsearch will be splitted to chunks according given byte limit. Default value 1000000. |
 | output.good.chunk.recordLimit | Optional. Bulk request to Elasticsearch will be splitted to chunks according given record limit. Default value 500. |
-| output.bad.type | Required. Configure where to write bad rows. Can be "kinesis", "nsq", "stderr" or "none". |
+| output.bad.type | Required. Configure where to write failed events. Can be "kinesis", "nsq", "stderr" or "none". |
```

Comment:
We will need to adjust this for Enrich 5.0.0 because we will have 2 streams of failed events. I prefer to say they are both failed events in 2 different formats, rather than say there are bad rows and there are failed events (or there are bad rows and there are incomplete events).

But for now just making it uniform.

Reply:
`output.incomplete.*` is on its way, I'll prepare the docs next week, how should we differentiate them ? Failed events for the warehouse and failed events for storage ?